packages/atomic/storybook-utils/common/render-component.tsx

```diff
@@ -27,6 +27,10 @@ const renderComponentInternal =
         case 'slots':
           slotsControls.push(argKey);
           break;
+        default:
+          if (argKey.startsWith('attributes-depends-on-')) {
```

Comment:
Since the `attributes-depends-on` with the postfix isn't in the table in the `context` object (i.e., `context.argTyppes[attributes-depends-on-foo].table` will always be undefined), we have to deal with those differently.